src/09_dictionaries.py

```diff
@@ -35,10 +35,13 @@
 
 # Add a new waypoint to the list
 # YOUR CODE HERE
-
+waypoints.append({"lat": 10, "lon": 55, "name": "an additional spot"})
 # Modify the dictionary with name "a place" such that its longitude
 # value is -130 and change its name to "not a real place"
 # YOUR CODE HERE
-
+waypoints[0] = {"lat": -130, "lon": 55, "name": "not a real place"}
 # Write a loop that prints out all the field values for all the waypoints
-# YOUR CODE HERE
\ No newline at end of file
+# YOUR CODE HERE
+for i in range(len(waypoints)):
```

Comment:
another way to do this is:

`for i in waypoints:
    print(i['name'], i['lat'], i['lon'])`